lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -140,6 +148,8 @@ class DataDownload
         dataType: 'json'
         url: url
         error: (std_ajax_err) =>
+          if std_ajax_err.status == 401
+            window.location.reload()
```

Comment:
@dan-f Because we can't handle redirect in javascript, browser always follow the redirected url but we need to check if response.status is 302 then reload the ```window.location```.